csharp/src/elements/Geometry/Polygon.cs

```diff
@@ -140,31 +325,124 @@ public bool Disjoint(Polygon polygon)
         }
 
         /// <summary>
-        /// Tests if this Polygon and the supplied Polygon share areas within their perimeters.
+        /// Tests if any of the supplied Polygons are coincident with this Polygon.
         /// </summary>
-        /// <param name="polygon">The Polygon to compare.</param>
+        /// <param name="polygons">The collection of Polygons to compare.</param>
         /// <returns>
-        /// Returns true if any vertex of either Polygon is within the perimeter of the other.
+        /// Returns true if any of the supplied Polygons do not intersect or touch.
         /// </returns>
-        public bool Intersects(Polygon polygon)
+        public bool Disjoint(IList<Polygon> polygons)
```

Comment:
As above, use singular `Disjoint` in this method.

Reply:
Done